src/BehaviorsSDKNative/Microsoft.Xaml.Interactions/Resources/uk-UA/Strings.resw

```diff
@@ -1,110 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
```

Comment:
Was this change generated? Same for the other Strings.resw under Interactivity

Reply:
Yes, this file is generated. I'm guessing the toolkit now outputs that commented text at the top.